ds_judgements_public_ui/templates/pages/accessibility_statement.html

```diff
@@ -105,17 +105,17 @@ <h2 id="section-contact">Contacting us or visiting in person</h2>
             chat. We provide information about
             <a href="https://www.nationalarchives.gov.uk/about/visit-us/">visiting in person</a>.
             People with health conditions or impairments that may impact their visit can
-            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more</a>.
+            <a href="https://www.nationalarchives.gov.uk/about/visit-us/information-for-disabled-visitors/">find out more about access building details</a>.
```

Comment:
The phrase "find out more about access building details" doesn't track - should it be "accessibility details"?

Reply:
@Terry-Price please see @jacksonj04 comment images of before and after text changes
before
![before-text](https://github.com/user-attachments/assets/18be41a8-0f4c-4f5c-8920-cdf60b6f6171)
after
![after-text](https://github.com/user-attachments/assets/01c4cf7a-4f4e-416e-a075-cb067343d9a7)

Reply:
Hi @Donna-H 
Do you mind changing the date at the top so it doesnt use 'th'.

So 'DD Month YYYY'

Reply:
can we maybe change 'find out more about access building details' to ' find out more how we provide full access to all our public areas'  as the first line states on TNA website below ? @jacksonj04 @Terry-Price ![new-text](https://github.com/user-attachments/assets/8c635540-d182-44c0-8638-efa53026281a)

Reply:
Is it as straightforward as should be "building access details" not "access building details"? That would track!

Reply:
@Terry-Price this short form can be retained then by switching order of text as @jacksonj04 suggested is that okay ?